### PR TITLE
Fix magma script issues.

### LINKED ISSUE #7374: Vulnerability in math.js used in MagmaScript
### How to Reproduce
Check version of [math.js](https://github.com/molgenis/molgenis/blob/master/molgenis-js/src/main/resources/js/math.min.js)

### Expected behavior
It is up to date

### Observed behavior
It is an old version [with security issues](https://nodesecurity.io/advisories?search=mathjs&version=2.1.1).

### PR DESCRIPTION
**Pull request by fdlk**
Two commits.
edd64d8 is the smallest fix for the val -> value() issue, can be cherry picked to 6.1.
07ce8db fixes #7374 
#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
- [x] User documentation updated
- [x] (If you have changed REST API interface) view-swagger.ftl updated
- [x] Test plan template updated
- [x] Clean commits
- [x] Added Feature/Fix to release notes

### CODE REVIEW

File: docs/user_documentation/ref-expressions.md [hunk 1]
```diff
@@ -47,17 +47,17 @@ $('myIntAttributeName').gt(3).and($('myIntAttributeName').lt(6)).value()
...
-| map      | Object     | $('data').map({0:1, 1:2}).value()        | Maps categories to eachother                                                               |
-| group    | Array      | $('age').group([18, 35, 50, 75]).value() | Produces ranges which are left inclusive, (-∞, 18), [18, 35), [35, 50), [50, 75), [75, +∞)|                                                                |
+| Operator | Parameters | Example                                  | Description                                                           |
+|----------|------------|------------------------------------------|-----------------------------------------------------------------------|
+| age      | -          | $('dateOfBirth').age().value()           | Returns the age based on the date of birth and the current year       |
+| map      | Object     | $('data').map({0:1, 1:2}).value()        | Maps categories to each other                                         |
+| group    | Array      | $('age').group([18, 35, 50, 75]).value() | '18-35'                                                               |
```
Comment (dennishendriksen) [inline docs/user_documentation/ref-expressions.md]: The previous description is more understandable for me.

File: molgenis-js/src/main/resources/js/math.min.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+"use strict";
```
Comment (dennishendriksen) [inline molgenis-js/src/main/resources/js/math.min.js]: math.js [4.3.0](http://mathjs.org/history.html) has breaking changes compared to 2.1.1. Is this desirable?

File: docs/user_documentation/ref-expressions.md (hunk 1 above)
Comment (fdlk) [reply docs/user_documentation/ref-expressions.md]: it is also incorrect. shall I merge them?

File: molgenis-js/src/main/resources/js/math.min.js (hunk 2 above)
Comment (fdlk) [reply molgenis-js/src/main/resources/js/math.min.js]: I strongly suspect it is only in use indirectly by MagmaScript unit conversions. Updating release notes.

**Pull request merged**